docs/milestones.md

```diff
@@ -0,0 +1,21 @@
+
+## [M1] - Registro y Visualización de Gastos Fijos
+Entregable: Código que permita registrar y visualizar los gastos fijos del estudiante, como el alquiler e internet, con un resumen claro del total comprometido en estos gastos (vinculado a HU1).  
+Viabilidad: Será viable cuando el sistema permita al estudiante registrar al menos dos tipos de gastos fijos (por ejemplo, alquiler e internet) y refleje su impacto en el presupuesto mensual. La viabilidad se valida cuando el sistema permite ingresar los gastos fijos y muestra el presupuesto restante después de haber deducido esos gastos, cumpliendo con los requisitos de la HU1, que demanda claridad sobre cuánto de su presupuesto está comprometido en gastos fijos. Las pruebas funcionales confirmarán que los gastos fijos se registran correctamente y que el presupuesto restante se actualiza correctamente.  
+
+## [M2] - Registro y Predicción de Gastos Variables
+Entregable: Código que permita registrar gastos variables (como ocio y facturas de electricidad) y generar predicciones basadas en datos históricos y estacionales sobre cómo estos gastos afectarán el presupuesto futuro del estudiante (vinculado a HU2).  
+Viabilidad: Será viable cuando el sistema permita registrar al menos tres tipos de gastos variables (por ejemplo, ocio, electricidad, transporte) y generar predicciones sobre cómo esos gastos afectarán al presupuesto futuro del estudiante. La viabilidad se valida cuando el sistema proporciona predicciones basadas en el comportamiento histórico de los gastos variables, lo que está relacionado con la HU2, que busca prever los gastos para permitir ajustes presupuestarios. Las pruebas deben confirmar que las predicciones de gastos variables son coherentes con los datos históricos y reflejan con precisión los cambios estacionales o patrones previos.  
+
+## [M3] - Planificación de Ahorro y Evaluación de Gastos Adicionales
+Entregable: Código que permita configurar un plan de ahorro mensual y evaluar la viabilidad de realizar un gasto adicional, mostrando su impacto en el presupuesto restante del estudiante (vinculado a HU3 y HU4).  
+Viabilidad: Será viable cuando el sistema permita al estudiante configurar un plan de ahorro mensual y evaluar la viabilidad de realizar un gasto adicional basado en su presupuesto actual. La viabilidad se valida cuando el sistema calcula si el gasto adicional es viable sin comprometer las metas de ahorro, lo cual está relacionado con HU3 y HU4, que requieren que el estudiante pueda gestionar sus ahorros y decidir si un gasto adicional es financieramente viable. Las pruebas validarán que el sistema realiza correctamente los cálculos sobre el presupuesto restante después de registrar los gastos fijos y variables, y si el gasto adicional es viable sin afectar las metas de ahorro.  
```

Comment:
Un criterio de viabilidad de 8 líneas, ¿crees que es práctico? 
Poner tantos milestones, ¿crees de verdad que es práctico? ¿Has visto la planificación de la asignatura? ¿Cuando crees que podrás meter este milestone?
Si de verdad queréis meter milestones más allá del primero y segundo, mirad cómo están planificados los objetivos de la asignatura. Estás poniendo 3 milestones a base de características divididas arbitrariamente, cuando lo que hace falta es funcionalidad. ¿Has mirado qué tiene el objetivo 7, por ejemplo? ¿Y el 8?